kuka_ros2_control_sim_support/urdf/common_sim_macro.ros2_control.xacro

```diff
@@ -1,14 +1,31 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-  <xacro:macro name="kuka_common_ros2_control_macro" params="name prefix use_mock_hardware:=^|false">
+  <xacro:macro name="kuka_common_sim_ros2_control_macro" params="
```

Comment:
I would put this into `*_ros2_control_*` package and sim package can use that one.

Reply:
i put both files separately as we dont want traces of the gazebo in the  ros2_control only support package. anyways they have their default values set but still, for someone looking at ros2_control support package seeing gazebo related flags might be distracting, i thought @destogl